src/app/utils/translations/languages/French.json

```diff
@@ -61,12 +61,17 @@
   "minute": "minute",
   "minutes": "minutes",
 
-  "oppsNoResultsFound": "Oups, aucun résultat trouvé.",
+  "noResultsFound": "Oups, aucun résultat trouvé.",
   "trackNotAvailable": "Titre non disponible",
 
   "dataProvidedByGenius": "Données fournies par Genius",
   "inYourPersonalLibrary": "Dans votre bibliothèque personnelle",
   "unauthorisedLibrarySearch": "Vous devez vous connecter pour rechercher votre bibliothèque.",
 
-  "noLyricsAvailable": "Pas de paroles disponibles"
+  "noLyricsAvailable": "Pas de paroles disponibles",
```

Comment:
I noticed before that we don't have a "fallback" language. I think that unless you add the new strings to each locale file then it shows up as "undefined" in other regions. Will need to confirm.

Reply:
It does fallback, we had some misnamed french translation and it fallbacks to english